src/applications/appeals/testing/sc/content/evidenceVaRecords.jsx

```diff
@@ -4,18 +4,23 @@ import numberToWords from 'platform/forms-system/src/js/utilities/data/numberToW
 
 export const content = {
   title: (addOrEdit, index) =>
-    `${addOrEdit === 'add' ? 'Add' : 'Edit'} the ${numberToWords(
+    `${addOrEdit === 'add' ? 'Add a' : 'Edit the'} ${numberToWords(
       index || 1,
-    )} VA facility that treated you`,
-  description: 'We’ll request your VA medical records from this facility',
-  locationAndName:
-    'Name of VA medical center, VA treatment facility, or Federal department or agency',
+    )} VA or military treatment location`,
+  description:
+    'We’ll request your VA medical records from this facility or provider',
+  locationAndName: 'Name of facility or provider that treated you',
+  locationAndNameHint:
+    'You can enter the name of a VA medical center or clinic, Vet Center,  military treatment facility, or community care provider',
 
-  issuesLabel:
-    'Choose the conditions you received treatment for at this facility.',
-  dateStart: 'First treatment date (you can estimate)',
-  dateEnd: 'Last treatment date (you can estimate)',
-  addAnotherLink: 'Add another location',
+  issuesLabel: 'What did they treat you for?',
+  // dateStart: 'First treatment date (you can estimate)',
+  treatmentDate: 'When did they treat you?',
+  treatmentHint:
+    'If treatment began from 2005 to present, you do not need to provide date. ',
```

Comment:
Is the trailing space needed?

Reply:
Or in this one since there are now merge conflicts LOL